### PR TITLE
Jump to: Channels and Users Navigation

### PR DESCRIPTION
**Pull request by nzikak**
**Jump To Channels And Users Navigation**
Added Navigation to the Jump To Page in order to navigate to channels and DMs of users selected. The DM page still uses dummy data so clicking on a user from the page doesn't actually start a DM with the selected user. I'll change it to send real user data to the DM page once that becomes fully operational but navigation to a channel works correctly.

**Demo Video**

https://user-images.githubusercontent.com/31522978/136440957-34437278-2220-43d1-a85a-ad6a55fd4370.mp4

### CODE REVIEW

File: lib/general_widgets/custom_channel.dart [hunk 1]
```diff
@@ -11,37 +11,34 @@ class CustomChannel extends StatelessWidget {
...
+              child: lock == true ?
+              SvgPicture.asset(
+                  'assets/icons/svg_icons/lock_outline.svg',
+              ) :
+              SvgPicture.asset(
+                'assets/icons/svg_icons/hash_tag.svg',
+              )
+            ),
+            // Text(
+            //   '#',
```
Comment (jeremiahseun) [inline lib/general_widgets/custom_channel.dart:33]: Please remove unused commented code

File: lib/ui/view/dm_chat_view/dm_jump_to_view.dart [hunk 2]
```diff
@@ -118,10 +123,19 @@ class DmJumpToView extends StatelessWidget with $DmJumpToView {
...
                             itemBuilder: (context, i) {
-                              return CustomChannel(
-                                text: model.allChannelsSearch[i].name ??
-                                    ChannelName,
-                                // lock: model.allChannelsSearch[i].private== "True",
+                              final channel = model.allChannelsSearch[i];
+                              return InkWell(
+                                child: CustomChannel(
+                                  text: channel.name ?? ChannelName,
+                                  // lock: model.allChannelsSearch[i].private== "True",
```
Comment (jeremiahseun) [inline lib/ui/view/dm_chat_view/dm_jump_to_view.dart:130]: Will you need it later?

**Review by jeremiahseun — CHANGES_REQUESTED** [review]
Please make changes

File: lib/general_widgets/custom_channel.dart (hunk 1 above)
Comment (nzikak) [reply lib/general_widgets/custom_channel.dart]: I didn't comment the code but I've removed the comments

File: lib/general_widgets/custom_user.dart [hunk 3]
```diff
@@ -9,36 +9,33 @@ class CustomUser extends StatelessWidget {
...
+        mainAxisAlignment: MainAxisAlignment.start,
+        children: [
+          Container(
+            height: 48.h,
+            decoration: BoxDecoration(
+                borderRadius: BorderRadius.circular(8.r),
+                image: const DecorationImage(
+                    fit: BoxFit.cover,
+                    image: NetworkImage(
+                        'https://www.nairaland.com/attachments/10525387_7523740813649122436786583150820259255025664o3_jpega51895fd0bf50cf5d5982826ac401627'))),
```
Comment (pauleke65) [inline lib/general_widgets/custom_user.dart:24]: Add this to app_strings.dart

Comment (pauleke65) [inline lib/general_widgets/custom_user.dart:24]: Add this to app_strings.dart

File: lib/ui/view/dm_chat_view/dm_jump_to_view.dart [hunk 4]
```diff
@@ -40,7 +40,7 @@ class DmJumpToView extends StatelessWidget with $DmJumpToView {
                 crossAxisAlignment: CrossAxisAlignment.start,
                 children: [
                   SizedBox(
-                    height: 40.h,
+                    height: 45.h,
                     width: 361.w,
                     child: TextField(
```
Comment (pauleke65) [inline lib/ui/view/dm_chat_view/dm_jump_to_view.dart:45]: Add 
```textAlignVertical: TextAlignVertical.center```
To the TextField Widget to align the text properly

**Review by jeremiahseun — APPROVED** [review]
LGTM

**Pull request merged**